pages/platform/kubernetes-k8s/label-taint-node-pools/guide.en-asia.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Deploy applications to specific Nodes and Nodes Pools
+slug: label-nodeaffinity-node-pools
+excerpt: 'Find out how to deploy applications to specific Nodes and Nodes Pools, with labels and NodeAffinity, on OVHcloud Managed Kubernetes'
+section: Tutorials
+---
+
+<style>
+ pre {
+     font-size: 14px;
+ }
+ pre.console {
+   background-color: #300A24;
+   color: #ccc;
+   font-family: monospace;
+   padding: 5px;
+   margin-bottom: 5px;
+ }
+ pre.console code {
+   border: solid 0px transparent;
+   font-family: monospace !important;
+   font-size: 0.75em;
+   color: #ccc;
+ }
+ .small {
+     font-size: 0.75em;
+ }
+</style>
+
+**Last updated 15 December 2021.**
+
+## Objective
+
+In this tutorial we are going to show you how to deploy your applications to specific `Nodes` and` Nodes Pools`, with `labels` and `NodeAffinity` Kubernetes concepts, on your OVHcloud Managed Kubernetes Service.
+
+We will:
+
+- create a Managed Kubernetes cluster
+- create a node pool with 3 nodes and "monthly" billing mode
+- create another node pool with autoscaling activated (0 nodes minimum and maximum 10), with "hourly" billing mode
+- deploy an application to a specific node pool (and nodes)
+
+
+## Requirements
+
+- a [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account
```

Comment:
Please update each website link according to the EN sub. For instance, here the link should be https://www.ovhcloud.com/asia/public-cloud/